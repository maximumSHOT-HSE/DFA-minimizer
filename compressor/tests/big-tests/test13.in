11 55 5
1 10 0
1 8 1
1 10 2
1 5 3
1 5 4
2 10 0
2 7 1
2 10 2
2 9 3
2 7 4
3 4 0
3 1 1
3 8 2
3 10 3
3 11 4
4 1 0
4 7 1
4 7 2
4 6 3
4 8 4
5 11 0
5 6 1
5 11 2
5 6 3
5 10 4
6 2 0
6 10 1
6 8 2
6 2 3
6 11 4
7 6 0
7 1 1
7 2 2
7 4 3
7 4 4
8 2 0
8 5 1
8 9 2
8 4 3
8 2 4
9 8 0
9 7 1
9 3 2
9 7 3
9 10 4
10 6 0
10 5 1
10 11 2
10 8 3
10 11 4
11 8 0
11 3 1
11 5 2
11 3 3
11 8 4
10
3
4 8 3 
