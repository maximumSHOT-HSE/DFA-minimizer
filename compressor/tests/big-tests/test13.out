11 55 5
4 1 0
3 1 1
7 1 1
6 2 0
8 2 0
7 2 2
6 2 3
8 2 4
11 3 1
9 3 2
11 3 3
3 4 0
7 4 3
8 4 3
7 4 4
8 5 1
10 5 1
11 5 2
1 5 3
1 5 4
7 6 0
10 6 0
5 6 1
4 6 3
5 6 3
2 7 1
4 7 1
9 7 1
4 7 2
9 7 3
2 7 4
9 8 0
11 8 0
1 8 1
3 8 2
6 8 2
10 8 3
4 8 4
11 8 4
8 9 2
2 9 3
1 10 0
2 10 0
6 10 1
1 10 2
2 10 2
3 10 3
5 10 4
9 10 4
5 11 0
5 11 2
10 11 2
3 11 4
6 11 4
10 11 4
10
3
3 4 8 
